NuGet.config

```diff
@@ -8,6 +8,7 @@
     <add key="vssdk" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vssdk/nuget/v3/index.json" />
     <!-- Required for	Microsoft.VisualStudio.Platform.VSEditor and Microsoft.VisualStudio.Text.Internal packages -->
     <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
+    <add key="vssdk-unifiedPIAs" value="https://pkgs.dev.azure.com/devdiv/DevDiv/_packaging/vssdk-unifiedPIAs/nuget/v3/index.json" />
```

Comment:
This is an internal feed hosting unsigned VS packages. Will remove once real VS build is running